pycbc/waveform/bank.py

```diff
@@ -341,7 +327,8 @@ def ensure_hash(self):
         self.table = self.table.add_fields(template_hash, 'template_hash')
 
     def write_to_hdf(self, filename, start_index=None, stop_index=None,
-                     force=False, skip_fields=None):
+                     force=False, skip_fields=None,
+                     write_compressed_waveforms=True):
```

Comment:
Add some documentation for the `write_compressed_waveforms` option to the function's doc string.